### ISSUE TITLE
MDI interface in subwindow mode allows for closing of measurement UI's

### ISSUE THREAD

**Issue by edbarnard** [ScopeFoundry/ScopeFoundry#9]
On macOs, the MDI subwindows for Measurements still have close button. The close button is still active and will permanently delete the Measurement's UI widget. We need to disable this close action or have a way to restore the UI.

**Comment by alanrbuckley** [comment]
Tried creating an event handler which supposedly can change the way the close signal is handled, but to no avail. PyQt reference is incredibly vague.

I was able to successfully load a new instance of the measurement module in the console by using 
```python
app.measurements.sine_wave_optimizer.__init__(app)
```
This creates its own new window which works properly.
However, the old module is still there. 

If we can add a pushButton which allows the user to close the old module in the measurement tree, it may serve as a temporary workaround while we work to resolve this issue.
